content/gophercon-2017/writing-a-successful-gophercon-proposal.md

```diff
@@ -0,0 +1,100 @@
++++
+author = ["David Cheney"]
+date = "2017-01-14"
+title = "Writing a successful GopherCon proposal"
+tags = ["gophercon","community"]
+series = ["GopherCon 2017"]
+
++++
+
+[The GopherCon 2017 Call for Proposals](https://www.papercall.io/gophercon2017) has reached the halfway mark so I wanted to pass on advice to potential speakers who are planning to submit a proposal but may be unsure how to go about it.
```

Comment:
> but may be unsure how to go about it.

This indicates we will be learning about **how** to submit a talk.. the following lines feel more like tips on **what** to include in a submission.. 

---
### Suggest:

[The GopherCon 2017 Call for Proposals](https://www.papercall.io/gophercon2017) has reached the halfway mark so I wanted to share some meaningful notes on what to consider while writing your application. Furthermore I wanted to encourage anyone thinking about submitting an application, to please do so.